mail_tracking_mass_mailing/hooks.py

```diff
@@ -3,26 +3,14 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 import logging
-from openerp import api, SUPERUSER_ID
+from openerp.addons.mail_tracking.hooks import column_add_with_value
```

Comment:
I'm thinking about adding the protection [directly in the `__init__.py` file of the addon in the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py), to skip these errors in the future more automatically.